rpcserver.go

```diff
@@ -3216,18 +3221,22 @@ func (r *rpcServer) sendPayment(stream *paymentStream) error {
 						PaymentError: err.Error(),
 						PaymentHash:  payIntent.rHash[:],
 					}); err != nil {
+						rpcsLog.Errorf("Failed "+
+							"sending on "+
+							"stream: %v", err)
+
 						select {
 						case errChan <- err:
-						case <-reqQuit:
-							return
+						default:
 						}
+						return
```

Comment:
IMHO it's a bit cleaner if the return is in the select scope. (Disregard my comment if this is common stlye in this code base.)

Reply:
Sorry, got confused here, this is OK, please discard my previous comment.